docs/source/howto/setup_computers.rst

```diff
@@ -0,0 +1,325 @@
+   It is a string among the following:
+
+   * ``RejectPolicy`` (default, recommended): reject the connection if the host is not known.
+   * ``WarningPolicy`` (*not* recommended): issue a warning if the host is not known.
+   * ``AutoAddPolicy`` (*not* recommended): automatically add the host key at the first connection to the host.
+
+Connection cooldown time (s)
+   The minimum time interval between consecutive connection openings to the remote machine.
+
+After setup and configuration have been completed, your computer is ready to go!
+
+.. important::
+
+   To check if you set up the computer correctly, execute:
+
+   .. code-block:: console
+
+      $ verdi computer test COMPUTERNAME
+
+   that will run a few tests (file copy, file retrieval, check of the jobs in the scheduler queue) to verify that everything works as expected.
+
+Keberos tokens
+--------------
+
+If the cluster you are using requires authentication through a Kerberos token (that you need to obtain before using ssh), you typically need to install ``libffi`` (``sudo apt-get install libffi-dev`` under Ubuntu), and make sure you install the ``ssh_kerberos`` optional dependencies during the installation process of AiiDA (see :ref:`intro:install:aiida-core`.
+Then, if your ``.ssh/config`` file is configured properly (in particular includes all the necessary ``GSSAPI`` options), ``verdi computer configure`` will contain already the correct suggestions for all the gss options needed to support Kerberos.
+
+Other commands for computers
+============================
+
+If you are not sure if your computer is already set up, use this command to get a list of existing computers:
+
+.. code-block:: console
+
+   $ verdi computer list
+
+To get detailed information on the specific computer named ``COMPUTERNAME``:
+
+.. code-block:: console
+
+   $ verdi computer show COMPUTERNAME
+
+To rename a computer or remove it from the database:
+
+.. code-block:: console
+
+   $ verdi computer rename OLDCOMPUTERNAME NEWCOMPUTERNAME
+   $ verdi computer delete COMPUTERNAME
+
+.. note::
+
+   You can delete computers **only if** no entry in the database is linked to them (as for instance ``CalcJob``, or ``RemoteData`` objects).
+   Otherwise, you will get an error message.
+
+It is possible to **disable** a computer.
+Doing so will prevent AiiDA from connecting to the given computer to check the state of calculations or to submit new calculations.
+This is particularly useful if, for instance, the computer is under maintenance but you still want to use AiiDA with other computers, or submit the calculations in the AiiDA database anyway.
+
+The relevant commands are:
+
+.. code-block:: console
+
+   $ verdi computer enable COMPUTERNAME
+   $ verdi computer disable COMPUTERNAME
+
+.. important::
+
+   The above commands will disable the computer for **all** AiiDA users.
+
+
+Limiting requests to the remote computer
+========================================
```

Comment:
This section is a duplicate of : https://444-77234579-gh.circle-artifacts.com/0/html/howto/installation.html#avoiding-overloads

Reply:
closing as changed after leo's merges